schema/crdb/dbinit.sql

```diff
@@ -2840,11 +2840,11 @@ CREATE TYPE IF NOT EXISTS omicron.public.switch_link_speed AS ENUM (
 );
 
 CREATE TABLE IF NOT EXISTS omicron.public.switch_port_settings_link_config (
-    port_settings_id UUID,
-    link_name TEXT,
-    mtu INT4,
+    port_settings_id UUID NOT NULL,
```

Comment:
These changes aren't strictly related to the FEC part of this PR.  These fields should always have been `NOT NULL`, and that oversight came to light when I realized that the `fec` field was already nullable.

Reply:
Is it possible to split this into two PRs? Marking the "rust side" of things as Nullable is an easy change to approve, but altering these columns to be non-NULL is a more involved change